app/models/dba/partition_all.rb

```diff
@@ -0,0 +1,71 @@
+class DBA::PartitionAll
+  TABLES = %w[
+    hmis_2020_affiliations hmis_2020_aggregated_enrollments
+    hmis_2020_aggregated_exits hmis_2020_assessment_questions
+    hmis_2020_assessment_results hmis_2020_assessments hmis_2020_clients
+    hmis_2020_current_living_situations hmis_2020_disabilities
+    hmis_2020_employment_educations hmis_2020_enrollment_cocs
+    hmis_2020_enrollments hmis_2020_events hmis_2020_exits hmis_2020_exports
+    hmis_2020_funders hmis_2020_health_and_dvs hmis_2020_income_benefits
+    hmis_2020_inventories hmis_2020_organizations hmis_2020_project_cocs
+    hmis_2020_projects hmis_2020_services hmis_2020_users
```

Comment:
We can ignore the 2020 versions since those are old and won't change at this point, and shouldn't be used other than for historic views.